cbor-file-magic.mkd

```diff
@@ -373,7 +373,8 @@ potential CoAP Content-Format number (examples are in {{ct-tags}}).
 ## Labeled CBOR Sequence Tag {#cbor-sequence-tag}
 
 IANA has allocated tag 55800 as the tag for the Labeled CBOR Sequence
-Enveloping Method.
+Enveloping Method from the CBOR Tags Registry.
```

Comment:
The introduction to 5 already says this, with an actual reference.

But maybe some more redundancy does not hurt.